One of the errors:

```
error[E0433]: failed to resolve: could not find `Builder` in `task`
   --> veilid-tools/src/spawn.rs:58:54
    |
58  |                     MustJoinHandle::new(tokio::task::Builder::new().name(name).spawn(future).unwrap())
    |                                                      ^^^^^^^ could not find `Builder` in `task`
    |
note: found an item that was configured out
   --> /build/cargo-vendor-dir/tokio-1.39.2/src/task/mod.rs:368:26
    |
368 |         pub use builder::Builder;
    |                          ^^^^^^^
help: consider importing one of these items
    |
```

--- a/pkgs/tools/networking/veilid/default.nix
+++ b/pkgs/tools/networking/veilid/default.nix
@@ -6,17 +6,18 @@
 , rustPlatform
 , protobuf
 , capnproto
+, cmake
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "veilid";
-  version = "0.3.3";
+  version = "0.3.4";
 
   src = fetchFromGitLab {
     owner = "veilid";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-Gm65fvLImbsAU8kMYQv5VFEjkBQnhBFDqwheddRbtU8=";
+    hash = "sha256-nEJxiox2aoQBV83vlpiBB4In59+lfHF6/a8HqDYcFT4=";
   };
 
   cargoLock = {
@@ -31,6 +32,7 @@ rustPlatform.buildRustPackage rec {
 
   nativeBuildInputs = [
     capnproto
+    cmake
     protobuf
   ];
 
@@ -39,6 +41,8 @@ rustPlatform.buildRustPackage rec {
   cargoBuildFlags = [
     "--workspace"
   ];
+
+  RUSTFLAGS = "--cfg tokio_unstable";
 
   doCheck = false;
 

